bolt-sidecar/src/builder/payload_builder.rs

```diff
@@ -243,6 +227,42 @@ impl FallbackPayloadBuilder {
             i += 1;
         }
     }
+
+    /// Fetch the previous RANDAO value from the beacon chain.
+    ///
+    /// NOTE: for some reason, using the ApiResult from `beacon_api_client` doesn't work, so
+    /// we are making a direct request to the beacon client endpoint.
```

Comment:
hmm just curious to know some more detail

Reply:
I think it has to do with the type that the `beacon_api_client` crate expects to deserialize from the result of that API call - maybe it has changed or it's not updated to what Lighthouse is responding, so it fails to deserialize.

This is probably a bug in the beacon_api_client crate - but I haven't investigated